src/components/Testinho/Testinho.stories.tsx

```diff
@@ -0,0 +1,5 @@
+import type { Story } from '@ladle/react';
+
+import Testinho from './Testinho';
+
+export const World: Story = () => <Testinho />;
```

Comment:
Aqui precisa ser o nome do componente

```suggestion
export const Testinho: Story = () => <Testinho />;
```